kubernetes/charts/weblogic-operator/templates/_operator-dep.tpl

```diff
@@ -58,8 +58,15 @@ spec:
         {{- end }}
         resources:
           requests:
-            cpu: "100m"
-            memory: "512Mi"
+            cpu: {{ .cpuRequests | default "100m" }}
+            memory: {{ .memoryRequests | default "512Mi" }}
+          limits:
```

Comment:
Should the defaults be increased?  This question is for both Anil & @rjeberhard

Reply:
Yes, I agree that defaults should be increased at least for cpu request. With cpu request of "100m", JVM uses "defNew" and "SerialOld" GC (single threaded). Even when I configured JVM to use G1 GC, JVM ran with a parallel GC but with only one GC thread. However, I'm not aware of reason for previous default value of "100m". I wanted to get @rjeberhard Ryan's thoughts on the default values and I can increase them accordingly.